service/RTC/MediaType.js

```diff
@@ -1,15 +1,9 @@
 /**
- * Enumeration of RTC media stream types
- * @type {{AUDIO: string, VIDEO: string}}
+ * The audio type.
  */
-var MediaType = {
-    /**
-     * The audio type.
-     */
-    AUDIO: "audio",
-    /**
-     * The video type.
-     */
-    VIDEO: "video"
-};
-module.exports = MediaType;
\ No newline at end of file
+export const AUDIO = "audio";
```

Comment:
Sounds interesting, but that's too much for this PR. I could break something for sure